packages/babel-plugin-transform-inline-environment-variables/__tests__/inline-env-var-test.js

```diff
@@ -20,4 +20,28 @@ describe("inline-env-plugin", () => {
 
     process.env.NODE_ENV = prev;
   });
+
+  it("should only include whitelisted variables if include option is specified", () => {
+    const prev = process.env.NODE_ENV;
+    process.env.NODE_ENV = "development";
+    const include = ["IS_ELECTRON"];
+
+    const source = "process.env.NODE_ENV";
+    const expected = "process.env.NODE_ENV;";
+    expect(transform(source, { include })).toBe(expected);
```

Comment:
the expected value should be "development" here. Also can you please add process.env.IS_ELECTRON to the source in the test.